lib/batch/subject.csv.rb

```diff
@@ -0,0 +1,22 @@
+csv = CSV.open('/home/deploy/all_gf_subject.csv', 'wb')
+terms =  GalterGenericFilePresenter.terms + [:depositor, :date_uploaded, :date_modified, :read_access_group]
+
+csv << terms.map(&:to_s).map {|t| I18n.t(:simple_form)[:labels][:generic_file][t] || t.to_s.titleize }
```

Comment:
Let's change `|t|` to `|term|` or `|e|`? `t` is used for `I18n.translate` and that is unnecessarily confusing.

Reply:
Ha, it is. I will change to `term` and `I18n.translate`.